### PR TITLE
feat: Change uvicorn to flask

### PR DESCRIPTION
**Pull request by JakWai01**
Closes https://github.com/alphahorizonio/consumat.io/issues/14

### CODE REVIEW

**Comment by r2binx** [comment]
LGTM but this change should also be reflected in the readme and requirements files.

**Review by pojntfx — CHANGES_REQUESTED** [review]
Almost works, but requires two more things:

- The port _must_ be configurable with the `PORT` environment variable. This is a requirement by Heroku, see the [Procfile](./Procfile)
- Please also update the Procfile accordingly. You can manually set the PORT env variable on your local system, and then run the command after `web: ` in the Procfile to test.

File: Procfile [hunk 1]
```diff
@@ -1 +1 @@
-web: uvicorn consumatio.external.api:app --host=0.0.0.0 --port=${PORT:-5000}
+web: python3 api.py --host=0.0.0.0 --port=${PORT:-5000}
```
Comment (pojntfx) [inline Procfile:1]: `api.py` isn't at the root directory - also, shouldn't `--host` and `--port` be dropped? Additionally, please set the `HOST` variable here (`HOST=0.0.0.0 python3`), Heroku only sets `PORT`.

File: consumatio/external/api.py [hunk 2]
```diff
@@ -130,5 +131,8 @@ def graphql_server():
     status_code = 200 if success else 400
     return jsonify(result), status_code
 
+host = os.environ['HOST']
+port = os.environ['PORT']
```
Comment (pojntfx) [inline consumatio/external/api.py:135]: What happens if port is not an integer?

**Review by pojntfx — CHANGES_REQUESTED** [review]

File: consumatio/external/api.py (hunk 2 above)
Comment (pojntfx) [reply consumatio/external/api.py:135]: Port casting added.

**Review by pojntfx — APPROVED** [review]
LGTM!

File: Procfile (hunk 1 above)
Comment (pojntfx) [reply Procfile]: See following commit.

**Pull request merged**